Casks/chromedriver.rb

```diff
@@ -1,14 +1,18 @@
 cask "chromedriver" do
-  version "87.0.4280.20"
-  sha256 "99e98317a5f883d53973a044778f81abe71b42c570fb35f75786f86ea617bd1a"
+  version "87.0.4280.88"
+  sha256 "e8e4b2eaaf3e9433e368c125643973f5fabeedff349953014ab99c622ed61011"
 
   url "https://chromedriver.storage.googleapis.com/#{version}/chromedriver_mac64.zip",
       verified: "chromedriver.storage.googleapis.com/"
-  appcast "https://chromedriver.storage.googleapis.com/LATEST_RELEASE"
   name "ChromeDriver"
   desc "Automated testing of webapps for Google Chrome"
   homepage "https://sites.google.com/a/chromium.org/chromedriver/home"
 
+  livecheck do
+    url "https://chromedriver.storage.googleapis.com/LATEST_RELEASE"
+    regex(/.*/)
```

Comment:
```suggestion
    regex(/v?(\d+(?:\.\d+)+)/i)
```

Unless the looseness is necessary, we should bring this more in line with `charles`/`chromium`.